bundle.js

```diff
@@ -0,0 +1,105 @@
+(() => {
```

Comment:
bundle js is automatic and therefore generally done well, can see scope of how project uses javascript and html